### ISSUE TITLE
Error_log Warning: Illegal string offset 'url'

### ISSUE THREAD

**Issue by petersgallery** [dudaster/ele-custom-skin#469]
Been getting a lot of errors in the log (screenshot). I have a live auction at the moment and the site is dragging its feet. It is way too slow and i'm trying to solve this hoping it will help the situation. 

I read this page https://www.howtoonlinetips.com/elementor-warning-controls-stack-php/ and was considering changing the control-stacks.php file at line 1159 (as mentioned in the error log) to stop the issue. But i have no idea what i'm doing so,

Please advise on how to resolve this?

Elementor Pro Support says: "... contact Ele Custom Skin plugin support as they are in a better position to assist you with this"

<img width="752" alt="Screenshot 2021-04-01 at 00 31 23" src="https://user-images.githubusercontent.com/81747623/113275379-68e6f800-92e7-11eb-8215-37b9844caa0d.png">
<img width="912" alt="Screenshot 2021-04-01 at 00 32 44" src="https://user-images.githubusercontent.com/81747623/113275391-6d131580-92e7-11eb-9b6f-274fee3f27b9.png">

**Comment by mmg68** [comment]
Having the same exact issue. @petersgallery were you able to figure out a solution? @dudaster any help would be much appreciated.

**Comment by mmg68** [comment]
I found this solution and it worked for me. https://github.com/dudaster/ele-custom-skin/issues/118 .

**Comment by mmg68** [comment]
@dudaster just did update to current version of Elementor 3.2.3 and when I tried to update the Elementor plugin as described in the post I mentioned above (#118 ), it seems the code on line 1159 elementor/includes/base/control-stack the code has changed. The code has changed from:
 
`if ( $dynamic_property ) {`  

it now is  

`$active_settings[ $setting_key ] = $setting;
			} else {`

Not sure this solution will work without breaking something else.  Any suggestions - I'm getting ready to move this project live and any direction would be much appreciated. Is this Warning: Illegal string offset 'url' an issue in the Pro version of Ele Custom Skin?

**Comment by dudaster** [comment]
The question is it is still throwing an error?

**Comment by mmg68** [comment]
Yes, you can view it on my staging site

**Comment by dudaster** [comment]
Are you using make column clickable plugin?

**Comment by mmg68** [comment]
Yes, I'm using Steroids for Elementor. Is that the culprit? I'll try to remove and see if it removes error. Thank you for quick response!

**Comment by mmg68** [comment]
Yup, that was it. Again, really appreciate your help.
